pkg/component/registrycaches/registry_caches_test.go

```diff
@@ -238,7 +238,37 @@ proxy:
 				return config
 			}
 
-			statefulSetFor = func(name, upstream, size, configSecretName, tlsSecretName, tlsSecretChecksum string, storageClassName *string, additionalEnvs []corev1.EnvVar) *appsv1.StatefulSet {
+			_ = func(name, upstream, remoteURL string) *corev1.Service {
+				return &corev1.Service{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      name,
+						Namespace: "kube-system",
+						Labels: map[string]string{
+							"app":           name,
+							"upstream-host": upstream,
+						},
+						Annotations: map[string]string{
+							"upstream":   upstream,
+							"remote-url": remoteURL,
+						},
+					},
+					Spec: corev1.ServiceSpec{
+						Selector: map[string]string{
+							"app":           name,
+							"upstream-host": upstream,
+						},
+						Ports: []corev1.ServicePort{{
+							Name:       "registry-cache",
+							Port:       5000,
+							Protocol:   corev1.ProtocolTCP,
+							TargetPort: intstr.FromString("registry-cache"),
+						}},
+						Type: corev1.ServiceTypeClusterIP,
+					},
+				}
+			}
```

Comment:
These lines can be removed as registry cache services are being moved to a separate component and this function is now [here](https://github.com/gardener/gardener-extension-registry-cache/blob/853b68bf4c64f49126cb368d66fb8320b7aefa04/pkg/component/registrycacheservices/registry_cache_services_test.go#L60-L88).